client/src/App.jsx

```diff
@@ -1,20 +1,27 @@
 import "./App.css";
-
 import Layout from "./Layout";
-
 import Empty from "./components/Empty";
 import Entrance from "./components/Entrance";
 import Table from "./components/Table";
-
 import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { nanoid } from "nanoid";
+import axios from "axios";
 
 function App() {
+  const instanceID = nanoid();
+
+  async function getNewTable() {
+    await axios.post("/api/tables/new", {
+      ID: instanceID,
+    });
+  }
+
```

Comment:
I think this piece of code should be inside the <Entrance> component, since it's not being used anywhere else.